Scripts/datatypes/person.py

```diff
@@ -61,21 +61,11 @@ def calc_income(self):
 
     @property
     def area(self):
-        result = None
-        for area in areas:
-            if  areas[area][0] <= self.zone < areas[area][1]:
-                result = area
-                break
-        return result
+        return self.zone.area
 
     @property
     def municipality(self):
-        result = None
-        for mp in municipalities:
-            if  municipalities[mp][0] <= self.zone < municipalities[mp][1]:
-                result = mp
-                break
-        return result
+        return self.zone.municipality
```

Comment:
References to these can be replaced altogether by `self.zone.area` and `self.zone.municipality`, but I kept them for now to not break existing code.